src/api/ledger/accountinfo.js

```diff
@@ -6,7 +6,43 @@ const removeUndefined = require('./parse/utils').removeUndefined;
 const validate = utils.common.validate;
 const composeAsync = utils.common.composeAsync;
 
-function formatAccountInfo(response) {
+type AccountData = {
+  Sequence: number,
+  Account: string,
+  Balance: string,
+  Flags: number,
+  LedgerEntryType: string,
+  OwnerCount: number,
+  PreviousTxnID: string,
+  AccountTxnID: string,
+  PreviousTxnLgrSeq: number,
+  index: string
+}
+
+type AccountDataResponse = {
+  account_data: AccountData,
+  ledger_current_index: number,
```

Comment:
We want `ledgerVersion` to always be a number. Currently there is no support for lookups by ledger hash, but if we do add support for that, we should add a new field in the request called `ledgerHash`. I don't think there is any need to specify `validated` because that should be the default when no `ledgerVersion` is specified.